### ISSUE TITLE
Feature request: Install service file for D-Bus activation

### ISSUE THREAD

**Issue by ephemient** [Zren/plasma-hud#36]
```ini
# /usr/share/dbus-1/services/com.github.zren.PlasmaHUD.service
[D-BUS Service]
Name=com.github.zren.PlasmaHUD
Exec=/usr/lib/plasma-hud/plasma-hud
```

This will launch the binary on demand. XDG autostart would no longer be necessary.

**Comment by Zren** [comment]
When `krunner` migrated to dbus startup, the first use was slow. If `plasma-hud` is similar, it could cause the first few keystrokes into to get dropped if the user doesnt wait 1-2 seconds for the rofi window to appear.

I'll need to test this before adding the feature. It's python after all, so freeing up 40mb of RAM when not in use is a good idea. However, it's not worth ruining the performance over.

Also, if the service is always running, in the future I could pre-fetch the dbus menu before the user presses `Alt`.

**Comment by ephemient** [comment]
True, I do feel a slight delay on first launch, but it didn't feel that bad (on my admittedly well-powered machine). If that's a concern, it could still be autostarted, but with the service file, if the binary ever dies for some reason, this will kick it off again on demand instead of mysteriously disappearing.

**Comment by ephemient** [comment]
Also, as for existing `.desktop` file, it should contain

```ini
X-DBUS-ServiceName=com.github.zren.PlasmaHUD
```

because https://api.kde.org/frameworks/kservice/html/classKToolInvocation.html

> X-DBUS-StartupType=Unique Only start the service if it isn't already running, wait until the service has registered with D-Bus. The .desktop file can specify the name that the application will use when registering using X-DBUS-ServiceName=org.domain.mykapp. Otherwise org.kde.binaryname is assumed.
